invenio_rdm_records/contrib/thesis/custom_fields.py

```diff
@@ -26,7 +26,7 @@
     "section": _("Thesis"),
     "fields": [
         {
-            "field": "thesis:university",
+            "field": "thesis:thesis",
```

Comment:
ping @alejandromumo to verify, I am not sure what was the path in the end

Reply:
Indeed `thesis:university` was correct. It maps in many other places, we can't change it here I think:


https://github.com/search?q=org%3Ainveniosoftware%20%22thesis%3Auniversity%22&type=code
https://github.com/search?q=repo%3Azenodo%2Fzenodo-rdm%20%22thesis%3Auniversity%22&type=code

Reply:
I believe we can't change the custom field name now, even for migration would mean 💥 

However, if there's something wrong, I can help you out @0einstein0